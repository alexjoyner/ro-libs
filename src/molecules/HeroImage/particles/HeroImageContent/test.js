import React from 'react';
import { shallow } from 'enzyme';
import { HeroImageContent } from './';

describe('HeroImageContent component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<HeroImageContent />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
