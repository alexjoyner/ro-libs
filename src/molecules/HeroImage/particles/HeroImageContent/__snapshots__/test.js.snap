// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroImageContent component Should match snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    color: white;
",
        ],
      },
      "displayName": "styled.div",
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
